CONTRIBUTING.adoc

```diff
@@ -26,7 +26,7 @@ given the ability to merge pull requests.
 None of these is essential for a pull request, but they will all help.  They can also be
 added after the original pull request but before a merge.
 
-* Use the https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style[Spring Framework code format conventions].
+* Use the https://github.com/spring-projects/spring-framework/blob/master/CONTRIBUTING.md#source-code-style[Spring Framework code format conventions].
```

Comment:
[Unrelated to issue 983] I noticed that the original link did not go anywhere near the source code style page. However, the direct link actually references the `CONTRIBUTING.md` file in master path. I am happy to revert this but thought it may be helpful.